ghost/members-api/lib/repositories/member.js

```diff
@@ -439,6 +449,18 @@ module.exports = class MemberRepository {
             }
         }
 
+        // Throw error if email is invalid and it's been changed
+        if (
+            initialMember?.get('email') && memberData.email
+            && initialMember.get('email') !== memberData.email
+            && !validator.isEmail(memberData.email, {legacy: false})
+        ) {
+            throw new errors.ValidationError({
+                message: tpl(messages.invalidEmail),
+                property: 'email'
+            });
+        }
+
```

Comment:
Currently this won't validate the email address if we are not updating `products` or `newsletters`, because then initialMember will always be null 🤔

Reply:
Yeah true, I thought about that as an acceptable edge case given it won't be hit via the normal routes end user can update a member's email. Adding an extra check to fetch member and compare the email if it's changed seems like a big change to cover for this edge case atm, as the main idea is to block new invalid emails to enter the system. Wdyt ?

Reply:
Yes, I agree that now is not the right time to always fetch the member. Maybe we do need a cleanup issue for this?

Reply:
Yup, i'll add a note on original as well as to a related cleanup issue to update the email validator to use new method instead of legacy :)